core/src/main/java/com/amplifyframework/storage/StorageCategoryBehavior.java

```diff
@@ -201,6 +202,65 @@
         @NonNull Consumer<StorageUploadFileResult> onSuccess,
         @NonNull Consumer<StorageException> onError);
 
+    /**
```

Comment:
For any new APIs that get added to the `StorageCategoryBehavior`, we need to be sure to add the corresponding Rx versions to `RxStorageCategoryBehavior`, and associated implementation & tests.